e2e/framework/test_context.go

```diff
@@ -268,6 +268,10 @@ type OnpremConfig struct {
 	OS string `json:"os" yaml:"os" validate:"required,eq=ubuntu|eq=redhat|eq=centos|eq=debian"`
 	// DockerDevice block device for docker data - set to /dev/xvdb
 	DockerDevice string `json:"docker_device" yaml:"docker_device" validate:"required"`
+	// UsePublicAddress controls usage of public IP addresses for gravity site
+	// entry with terraform provider. In several situations(configured firewall) robotest
```

Comment:
You should mention what exactly adding flag does to fix the issues you describe - e.g. how does disabling the use of public IP address help and when would this be configured?